include/zephyr/net/wifi_mgmt.h

```diff
@@ -95,6 +95,8 @@ enum net_request_wifi_cmd {
 	NET_REQUEST_WIFI_CMD_AP_CONFIG_PARAM,
 	/** Flush PMKSA cache entries */
 	NET_REQUEST_WIFI_CMD_PMKSA_FLUSH,
+	/** Set enterprise mode file */
+	NET_REQUEST_WIFI_CMD_ENTERPRISE_FILE,
```

Comment:
`FILE` isn't the right term here, `ENTERPRISE_CREDS` or `CERT_INFO`?

Reply:
changed to ENTERPRISE_CREDS